thoth/storages/data/alembic/versions/fd56b28d89e6_create_enum_for_platform.py

```diff
@@ -0,0 +1,50 @@
+"""Create enum for platform
+
+Revision ID: fd56b28d89e6
+Revises: 50081eea5da2
+Create Date: 2021-06-15 13:52:56.057547+00:00
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+
+# revision identifiers, used by Alembic.
+revision = 'fd56b28d89e6'
+down_revision = '50081eea5da2'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.alter_column('depends_on', 'platform',
+               existing_type=sa.TEXT(),
+               type_=postgresql.ENUM('LINUX_X86_64', name='platform'),
+               existing_nullable=False,
+               existing_server_default=sa.text("'linux-x86_64'::text"))
+    op.alter_column('has_symbol', 'software_environment_id',
+               existing_type=sa.INTEGER(),
+               nullable=False)
+    op.alter_column('has_symbol', 'versioned_symbol_id',
+               existing_type=sa.INTEGER(),
+               nullable=False)
+    op.create_unique_constraint(None, 'python_package_version_entity_rule', ['id'])
```

Comment:
These three do not correspond to the PR. Might be some leftovers not captured in older PRs.

Reply:
Yes not sure why they appeared, I will update after I fix usual problems with using enums that are not automatically handled by migrations